src/QMCDrivers/WFOpt/EngineData.h

```diff
@@ -0,0 +1,32 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2019 QMCPACK developers.
```

Comment:
Minor issue - update the copyright year to 2022.

Reply:
Thanks, updated now.